src/api/util.js

```diff
@@ -2,6 +2,20 @@
  * Export util functions
```

Comment:
+1 Moving projection functions to the utils file but I think we can seize this opportunity to do a small refactor

Reply:
what kind of refactor?